views/help/confluence.hbs

```diff
@@ -4,16 +4,16 @@
     {{/content}}
 
     {{#content "scripts" mode="append"}}
-        <script src="/scripts/help.js" type="text/javascript" nonce="{{nonceValue}}" defer></script>
```

Comment:
Why should this be removed here?

Reply:
may be its not present in develop. it happened when merging!

Reply:
this needs to stay. it is present in develop.

Reply:
security related btw